bundles/org.openhab.binding.bluetooth.hdpowerview/README.md

```diff
@@ -0,0 +1,81 @@
+| pollingDelay            | Optional, Advanced | The scanning interval in seconds at which the binding checks the battery status (default 300 seconds).              |
+| encryptionKey           | Optional           | The key to be used when encrypting commands to the shade. See [next chapter](#encryption-key).                      |
+
+## Encryption Key
+
+If you want to send position commands to a shade, then an encryption key may be required.
+If the shade is NOT included in the Powerview App, then an encryption key is not required.
+But if it IS in the Powerview App, then openHAB has to use the same encryption key as used by the App.
+Currently you can only discover the encryption key by snooping the network traffic between the App and the shade.
+Please post on the openHAB community forum for advice about how to do this.
+
+## Channels
+
+A shade always implements a roller shutter channel `position` which controls the vertical position of the shade's (primary) rail.
+If the shade has slats or rotatable vanes, there is also a dimmer channel `tilt` which controls the slat / vane position.
+If it is a dual action (top-down plus bottom-up) shade, there is also a roller shutter channel `secondary` which controls the vertical position of the secondary rail.
+
+| Channel       | Item Type      | Description                                           |
+|---------------|----------------|-------------------------------------------------------|
+| position      | Rollershutter  | The vertical position of the shade's rail.            |
+| secondary     | Rollershutter  | The vertical position of the secondary rail (if any). |
+| tilt          | Dimmer         | The degree of opening of the slats or vanes (if any). |
+| battery-level | Number:Percent | Battery level (10% = low, 50% = medium, 100% = high). |
+| rssi          | Number:Power   | Received Signal Strength Indication.                  |
+
+Note: the channels `secondary` and `tilt` only exist if the shade physically supports such channels.
+
+## Examples
+
+```java
+// things
+Bridge bluetooth:bluegiga:abc123 "Bluetooth Stick" @ "Comms Cabinet" [port="COM3"] {
+    // shade NOT integrated in Powerview App
+    Thing bluetooth:shade:112233445566 "North Window Shade" @ "Office" [address="11:22:33:44:55:66"]
+
+    // or, shade integrated in Powerview App
+    Thing bluetooth:shade:112233445566 "North Window Shade" @ "Office" [address="11:22:33:44:55:66", encryptionKey="59409c980e627e2fc702c2efcbd4064d"]
+}
+
+// items
+Rollershutter Shade_Position "Shade Position" {channel="bluetooth:shade:abc123:112233445566:position"}
+Dimmer Shade_Position2 "Shade Position" {channel="bluetooth:shade:abc123:112233445566:position"}
+Dimmer Shade_Tilt "Shade Tilt" {channel="bluetooth:shade:abc123:112233445566:tilt"}
+Number:Percent Shade_Battery_Level "Shade Battery Level" {channel="bluetooth:shade:abc123:112233445566:battery-level"}
```

Comment:
`Number:Percent` is an unsupported type, i guess you mean `Number:Dimensionless`

Reply:
Ditto